File: Examples/CSharp/AsposePDF/DocumentConversion/PDFToEPUB.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.DocumentConversion
{
    public class PDFToEPUB
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_DocumentConversion();

            // load PDF document
            Document pdfDocument = new Document(dataDir + "PDFToEPUB.pdf");

            // instantiate Epub Save options
            EpubSaveOptions options = new EpubSaveOptions();
            
            // specify the layout for contents
            options.ContentRecognitionMode = EpubSaveOptions.RecognitionMode.Flow;
            
            // save the ePUB document
            pdfDocument.Save(dataDir + "Sample.epub", options);
        }
    }
}

File: Examples/VisualBasic/AsposePDF/DocumentConversion/PDFToEPUB.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf

Namespace AsposePDF.DocumentConversion
    Public Class PDFToEPUB
        Public Shared Sub Run()
            ' ExStart:PDFToEPUB
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_DocumentConversion()

            ' Load PDF document
            Dim pdfDocument As New Document(dataDir & Convert.ToString("PDFToEPUB.pdf"))

            ' Instantiate Epub Save options
            Dim options As New EpubSaveOptions()

            ' Specify the layout for contents
            options.ContentRecognitionMode = EpubSaveOptions.RecognitionMode.Flow

            ' Save the ePUB document
            pdfDocument.Save(dataDir & Convert.ToString("PDFToEPUB_out_.epub"), options)
            ' ExEnd:PDFToEPUB
        End Sub
    End Class
End Namespace